docs/consume-packages/Central-Package-Management.md

```diff
@@ -176,6 +176,28 @@ If you'd like to disable central package management for any a particular project
 </PropertyGroup>
 ```
 
+## Global Package References
```

Comment:
nit: Should the title be `GlobalPackageReference` to improve discoverability or?

Reply:
I think since its a title it is better to have it spelled out.  GlobalPackageReference is mentioned a few times in the body so it should show up in internet searches